maps/adhomai/code/structures/buildings.dm

```diff
@@ -0,0 +1,17 @@
+/obj/structure/girder/wood/adhomai
+	name = "wooden girder"
+	desc = "Gird your loins."
+	state = 0
+	health = 100
+	icon = 'icons/adhomai/structures.dmi'
+	icon_state = "windownew_frame"
+	/material/wood
+
+/obj/structure/girder/wood/dismantle()
+	new /obj/item/stack/material/wood(get_turf(src))
+	new /obj/item/stack/material/wood(get_turf(src))
+	qdel(src)
+
+/turf/simulated/wall/wood/adhomai
```

Comment:
what is this for.

Reply:
Deconstruction of the wooden girders. Otherwise it was producing metal because of how the code works.

Reply:
Needed.